todo/models.py

```diff
@@ -168,7 +174,7 @@ class Attachment(models.Model):
     Defines a generic file attachment for use in M2M relation with Task.
     """
 
-    task = models.ForeignKey(Task, on_delete=models.CASCADE)
+    task = models.ForeignKey(settings.TODO_TASK_MODEL, on_delete=models.CASCADE)
```

Comment:
I lost some time because of migration errors here. I didn't really know what was the appropriate field to give:
* `BaseTask` would have AttachmentAdmin complain that its task autocomplete field needs BaseTask register to admin (didn't make sense
* `"Task"` would have swapper complain that this model is swapped
* `settings.TODO_TASK_MODEL` would have the migration fail complaining that my local project's model didn't exist yet.

The 3rd option was correct, the issue was just regarding the order of migrations:
1. add todo to the project and migrate
2. create the model extending BaseTask and set the TODO_TASK_MODEL settings
3. makemigration and migrate

Normal users will not have any issue, since they'll run migrate while following the install documentation :+1: